dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/DefaultArguments.java

```diff
@@ -77,6 +84,47 @@ public String valuePattern() {
     }
   }
 
+  public static class ChronoUnitValueConverter implements ValueConverter<ChronoUnit> {
+
+    private enum AllowedUnits {
+      HOURLY(HOURS),
+      DAILY(DAYS),
+      WEEKLY(WEEKS),
+      MONTHLY(MONTHS),
+      YEARLY(YEARS);
```

Comment:
Can we keep the 'standard' names for the enum keys, rather than switching to a -LY form?